application/controllers/HostController.php

```diff
@@ -44,7 +44,7 @@ public function showAction()
 
             $this->params->add('name', $hostName);
 
-            if ($host !== false) {
```

Comment:
Are those changes really necessary?

Reply:
Yes, `$host` or `$service` will be `null` instead of `false` after restriction is applied.

Reply:
How can it return either `null` or `false`?

Reply:
I think there is a misunderstanding here. The query returns either the row as object or null (in case there is no result). So after the restriction the object will be null.

Reply:
So the check should be `!== null`, because `!== false` was never correct.

Reply:
Makes sense. I will make the changes.